### PR TITLE
Template improvements

### PR DESCRIPTION
**Pull request by emanuelschuetze**
- use modal dialogs for new/edit views of customslide/motions/assigments
- use hover actions in all list views

### CODE REVIEW

**Comment by normanjaeckel** [comment]
Ok (not tested).

File: openslides/agenda/static/js/agenda/site.js [hunk 1]
```diff
@@ -123,32 +101,43 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
...
+        $scope.editDialog = function (item) {
+            var template, controller, resolve;
+            switch (item.content_object.collection) {
+                case 'core/customslide':
+                    template = 'static/templates/core/customslide-form.html';
+                    controller = 'CustomslideUpdateCtrl';
+                    resolve = { customslide: function() {
+                        return item.getContentResource().find(item.content_object.id) }};
+                    break;
+                case 'motions/motion':
```
Comment (ostcar) [inline openslides/agenda/static/js/agenda/site.js]: Please find another way that does not need to use motion and assignment here

Comment (emanuelschuetze) [inline openslides/agenda/static/js/agenda/site.js]: Done

Comment (emanuelschuetze) [inline openslides/agenda/static/js/agenda/site.js]: @ostcar Now, the modal edit dialog opens correctly via ui-route. But also the "background page" is  redirected to this edit url. How can I prevent this (the page should keep on agenda.list)?

**Comment by emanuelschuetze** [comment]
@ostcar ping! I need this pull request for my next work on base template!

**Comment by emanuelschuetze** [comment]
rebased

**Pull request merged**